### PR TITLE
New package: meli-0.6.2

### PR DESCRIPTION
**Pull request by Logarithmus**
BSD/Linux terminal email client with support for multiple accounts and Maildir / mbox / notmuch / IMAP / JMAP.
See https://github.com/void-linux/void-packages/issues/26132

### CODE REVIEW

**Comment by Logarithmus** [comment]
Finally, after some dirty patching (the source code is such a mess, why would anyone use Linux syscalls directly in a mail client?!), CI check has been passed. Currently I have no time to test this package, so feel free to check out this branch, build it with usual `./xbps-src -C pkg meli | tee meli.log` and report the results.

File: srcpkgs/meli/patches/fixes-for-32bit-systems.patch [hunk 1]
```diff
@@ -0,0 +1,41 @@
+--- melib/src/backends/notmuch.rs
```
Comment (ericonr) [inline srcpkgs/meli/patches/fixes-for-32bit-systems.patch:1]: Have these been upstreamed?

**Comment by ericonr** [comment]
> Finally, after some dirty patching (the source code is such a mess, why would anyone use Linux syscalls directly in a mail client?!)

I'm not sure this makes me entirely optimistic about adding it as a new package :/

File: srcpkgs/meli/patches/fixes-for-32bit-systems.patch (hunk 1 above)
Comment (Logarithmus) [reply srcpkgs/meli/patches/fixes-for-32bit-systems.patch:1]: Not yet. It seems that the author doesn't know about the existence of 32-bit archs.

**Review by Logarithmus — COMMENTED** [review]
> > Finally, after some dirty patching (the source code is such a mess, why would anyone use Linux syscalls directly in a mail client?!)
> 
> I'm not sure this makes me entirely optimistic about adding it as a new package :/

I've decided to take a look at the requested packages and help with some of them.

**Comment by ericonr** [comment]
This is being closed on account of upstream doing sketchy things (which then require dirty patches on our side).

@Logarithmus, the patches should still be upstreamable, and someone can use this locally if they so wish. Thanks!

**Pull request closed without merging**